packages/webpack/lib/plugins/log.js

```diff
@@ -69,6 +69,15 @@ exports.LoggerPlugin = class LoggerPlugin {
           this.logger.info(formatWarning(name, duration, assets, isRebuild));
         } else {
           this.logger.info(formatSuccess(name, duration, assets, isRebuild));
+          const assetsOverSizeLimit = assets.some(
+            ({ isOverSizeLimit }) => isOverSizeLimit
+          );
+          if (assetsOverSizeLimit) {
+            this.logger.warn('Some assets exceed the recommended size limit!');
+            this.logger.hint(
+              'You can limit the size of your bundles by using our importComponent() helper to lazy load some parts of your application.'
```

Comment:
I would like to omit the usage of `yours` and `ours`.

Alternative: "You can limit the bundle size by using the importComponent() helper..."